sigopt_sklearn/search.py

```diff
@@ -257,7 +273,8 @@ def _fit(self, X, y, parameter_iterable=None):
         pre_dispatch = self.pre_dispatch
 
         # setup SigOpt experiment and run optimization
-        conn = sigopt.Connection(client_token=self.client_token)
+        conn = (self.sigopt_connection if self.sigopt_connection
```

Comment:
we should probably just put this in the constructor, and set self.sigopt_connection to either the connection they give us, or one we construct from the token